week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -6,9 +6,16 @@
   They have also decided that they should add a 1% fee to all foreign transactions
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
+function allForeign(price) {
```

Comment:
Good idea. You could have extended this idea to pass the rate as well and simply your two methods further.
 
```javascript
function allForeign(price, rate) {
    return parseFloat(price * rate + price /100)).toFixed(2);
}
```